src/Chapter3.hs

```diff
@@ -970,6 +1081,21 @@ implement the following functions:
 
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
+data DayOfWeek = Mon | Tue | Wed | Thu | Fri | Sat | Sun
+  deriving (Enum, Show)
+
+isWeekend :: DayOfWeek -> Bool
+isWeekend x = fromEnum x >= 5
```

Comment:
If you derive `Ord` for `DayOfWeek`, you can compare constructors directly, e.g. `x > Fri`. Though, it won't work if you decide to change the data type to the American system :slightly_smiling_face: